src/tactic/ui/panel/simple_side_bar_wdg.py

```diff
@@ -253,7 +253,23 @@ def get_link_wdg(my, element_name, config, options, info):
                 'link': link,
                 'target': target,
                 'cbjs_action': '''
-                var content = $(document).getElement(bvr.target);
+
```

Comment:
what are these . and # conditions for? It looks like css class selector.  Any usage example on how to define these simple side bar link that can be inserted here?